nservicebus/rabbitmq/configuration-api.md

```diff
@@ -187,9 +202,17 @@ In Versions 4 and above xml configuration options for controlling lost connectio
 snippet:rabbitmq-custom-breaker-settings-code
 
 
-### Changing routing topology
+### Routing topology
```

Comment:
That's not something I'd considered before. Seems like it could be useful to have something around this, but I think I'd want it in a different document.